HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.sch

```diff
@@ -2571,13 +2571,21 @@
       <sch:assert role='ERROR' test='count(h:WaterHeaterInsulation/h:Jacket/h:JacketRValue) &lt;= 1'>Expected 0 or 1 element(s) for xpath: WaterHeaterInsulation/Jacket/JacketRValue</sch:assert>
       <sch:assert role='ERROR' test='count(h:HotWaterTemperature) &lt;= 1'>Expected 0 or 1 element(s) for xpath: HotWaterTemperature</sch:assert>
       <sch:assert role='ERROR' test='count(h:UsesDesuperheater) &lt;= 1'>Expected 0 or 1 element(s) for xpath: UsesDesuperheater</sch:assert> <!-- See [Desuperheater] -->
+      <sch:assert role='ERROR' test='count(h:extension/h:HPWHInstalledProperly) &lt;= 1'>Expected 0 or 1 element(s) for xpath: HPWHInstalledProperly</sch:assert> <!-- See [HPWHInstalledProperly] -->
       <!-- Moved/deprecated extension/OperatingMode input; see https://github.com/NREL/OpenStudio-HPXML/pull/1289 -->
       <sch:assert role='ERROR' test='count(h:extension/h:OperatingMode) = 0'>extension/OperatingMode has been replaced by HPWHOperatingMode</sch:assert>
       <!-- Warnings -->
       <sch:report role='WARN' test='number(h:HotWaterTemperature) &lt; 110'>Hot water setpoint should typically be greater than or equal to 110 deg-F.</sch:report>
     </sch:rule>
   </sch:pattern>
 
+  <sch:pattern>
+    <sch:title>[HPWHInstalledProperly]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:WaterHeating/h:WaterHeatingSystem/h:extension[h:HPWHInstalledProperly="false"]'>
+      <sch:assert role='ERROR' test='count(h:HPWHContainmentVolume) = 1'>Expected 1 element(s) for xpath: HPWHContainmentVolume</sch:assert>
```

Comment:
Expected behaviors:

Installed Properly | Containment Volume | Reaction
-- | -- | --
TRUE | Provided | Ignore with warning
FALSE | Provided | Apply adjustment
TRUE | Missing | Pass without adjustment
FALSE | Missing | Error

Reply:
We decided to document the limitation instead of throwing warnings in this case.